corehq/apps/userreports/tasks.py

```diff
@@ -47,7 +47,7 @@ def _build_indicators(indicator_config_id, relevant_ids):
             # save is a noop if the filter doesn't match
             adapter.save(doc)
             redis_client.srem(redis_key, doc.get('_id'))
-        except DataError as e:
+        except Exception as e:
             logging.exception('problem saving document {} to table. {}'.format(doc['_id'], e))
```

Comment:
Do we get emails about `logging.exception`? I want to say no, and searching for "problem saving document" doesn't reveal anything except Sentry emails from before we killed sentry this summer. Aren't you worried that this isn't loud enough, or do you have some other notification source (like couchlog or something)?